spec/tool-bar-spec.js

```diff
@@ -76,9 +115,8 @@ describe('Tool Bar package', () => {
         toolBar = workspaceElement.querySelector('.tool-bar');
       });
 
-      it('by third-party packages', () => {
-        expect(toolBarAPI).toBeDefined();
-        expect(toolBarAPI.group).toBe('specs-tool-bar');
+      afterEach(() => {
+        toolBarAPI.unregister();
```

Comment:
And it does. IIRC/Jasmine calls `atom.reset`, which calls `@packages.reset()`, which in turn calls `@deactivatePackages()`.